.travis.yml

```diff
@@ -34,13 +34,11 @@ language: python
 
 matrix:
   include:
-    - python: 2.6
-      env: NUMPY=numpy==1.6 MOCK=mock
     - python: 2.7
-      env: MOCK=mock PANDAS=pandas
-    - python: 3.3
+      env: MOCK=mock NUMPY=numpy==1.6
```

Comment:
We should ask the numpy folks how many versions back they are supporting bug fixes for and don't go further back than that.

I am :+1: on dropping old versions of numpy, but that should be a separate discussion.